intro_to_rl.ipynb

```diff
@@ -0,0 +1,3225 @@
+{
```

Comment:
<p>I don't think its made clear why the if statement answer is wrong or bad. Pedagogically, it is far clearer than the jittable JAX example. At the very least, if you are going for jittable jax examples here, it should be made clear that this is expected. Personally, I would probably prefer just using numpy and standard python for this stuff, rather than forcing consideration of a second tricky concept (jittable jax).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>removed</p>